<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleDescription">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(230, 230, 230)</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="signup1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signup_password">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
padding-bottom:7px;
color:black;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signup_id">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
padding-bottom:7px;
color:black;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>User ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border:1px solid;
border-radius:15px;
color:black;
}

QPushButton:hover
{
background-color: white;

}
QPushButton:pressed
{
background-color:rgb(204, 204, 204);

}</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>true</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:black;}</string>
    </property>
    <property name="text">
     <string>Sign-up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="signup_confirm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
padding-bottom:7px;
color:black;</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Confirm-Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="signup_errormsg">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12px;
color: red;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
